bodhi/server/services/updates.py

```diff
@@ -92,6 +94,12 @@ def get_update(request):
                  error_handler=bodhi.server.services.errors.html_handler)
 def get_update_for_editing(request):
     """Return a single update from an id, title, or alias for the edit form"""
+    proxy_request = bodhi.server.security.ProtectedRequest(request)
+    validate_acls(proxy_request)
+    # If validate_acls produced 0 errors, then we can edit this update.
+    can_edit = len(proxy_request.errors) == 0
+    if not can_edit:
+        raise HTTPForbidden()
```

Comment:
okies, will look into this. Although i copied this from similar code up a bit higher in the same file:

https://github.com/ryanlerch/bodhi/blob/46271695395fbd1028a523f5244b0928f5bd5e7e/bodhi/server/services/updates.py#L88

So i guess we need to change it there too.